blog/_posts/2020-02-11-delete-clusters-faster-with-kind.md

```diff
@@ -0,0 +1,128 @@
+---
+slug: delete-clusters-faster-with-kind
+date: 2020-02-11
+author: nick
+layout: blog
+title: "Delete Your Kubernetes Clusters Faster with Kind"
+subtitle: "Announcing a Big Improvement in Local Development"
+image: "/assets/images/delete-clusters-faster-with-kind/priscilla-du-preez-acNPOikiDRw-unsplash.jpg"
+image_caption: "Photo by <a href='https://unsplash.com/@priscilladupreez?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText'>Priscilla Du Preez</a> on <a href='https://unsplash.com/s/photos/local?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText'>Unsplash</a>"
+tags:
+  - tilt
+keywords:
+  - tilt
+  - kind
+  - local development
+---
+
+Cluster API is a project to help you spin up new Kubernetes clusters with a
```

Comment:
There's something awkward and/or out of order in the first three paragraphs. The opening sentence isn't the main point of the post, and there's too much context laid out _before_ I'm hooked on the premise and decide to keep reading.

Suggestion:

One of the contributors of Kubernetes' Cluster API told us, no sugar-coating: "Tilt is good, and I'll let you finish, but Kind is one of the best Kubernetes dev tools of all time!” (Lightly paraphrasing.)

The reasoning was that since Cluster API is a project to help you spin up new Kubernetes clusters with a Kubernetes-style API, _the ability to quickly create and delete dev clusters is huge._

To start working, contributors create a fresh Kind cluster on their local machine, and they use Tilt to iterate on the project.

That in mind, my colleague Maia and I went to a Cluster API meeting in January to ask about the experience and what could be better, and [that's why we're excited to...]

Reply:
ya, I totally agree. i tried to rework it a bit. how's this?

Reply:
This is great! :)

Reply:
If I may nitpick just one more tiny bit... I would prefer to have the first sentence be even more of an active voice than passive. So, instead of:

> A few weeks ago, we talked to the Cluster API project to ask about their dev
experience and what would make it better.

I'd go with:

> A few weeks ago, we at Tilt asked the [Cluster API](link) team how to make their development experience even better.

Totally optional though, up to you.

Reply:
love it!